src/main/menu.ts

```diff
@@ -18,6 +18,13 @@ export default () =>
                         app.quit();
                     },
                 },
+                {
+                    label: '&Close',
```

Comment:
How about if we label this with “Close Window” and `CmdOrCtrl+Q` with “Quit nRF Connect for Desktop” to make the difference clearer? Maybe also ask Ketil whether he also has an idea about this?

Reply:
Good input, I like your suggestion, I will ask him as well